### DISCUSSION TITLE
DWMBlurGlass not hooking into DWM on latest Windows 11 Canary build (26052)

### DISCUSSION THREAD

**Discussion by raphm4044** [Maplespe/DWMBlurGlass#167] [Q&A]
**Describe the bug**
When installing DWMBlurGlass on Windows 11 24H2 (Canary build 26052), the hooking doesn't seems to works.

**To Reproduce**
Install it on W11 26052 (Canary Channel)

**Screenshot**
![image](https://github.com/Maplespe/DWMBlurGlass/assets/81994075/6aba83d2-ab8a-476a-b3b2-5238442fdeb2)

**Additional information (please complete the following information):**
OS Version : Windows 11 24H2 (build 26052 of Canary Channel)
Software Version : 2.0.1

**What other third-party software do you use:**
None

**Additional context**
It worked when i was on Windows 11, 23H2 stable release

**Comment by ALTaleX531** [comment]
> [!NOTE]
> Supported as low as Windows 10 2004 and as high as the latest version of Windows 11 (Some blurring methods are **not supported** in the preview version).

DwmBlurGlass has limited support for the preview version, and we can't guarantee that it can work properly.
